sdk/src/main/java/com/silanis/esl/sdk/builder/SignerBuilder.java

```diff
@@ -44,6 +44,7 @@
     private String placeholderName = null;
     private List<AttachmentRequirement> attachments = new ArrayList<AttachmentRequirement>();
     private KnowledgeBasedAuthentication knowledgeBasedAuthentication;
+    private String localLanguage;
```

Comment:
I have got the answer to my question about local language features. The local language is a type of language code, and it's strange to add an extra property to the signer model for this. In other words, it seems correct to add local as the value of a language attribute that exists at the current signer.

By default, the language priority is:

user language> signer language> package language

I think that the local language should go in one language code, not in a separate attribute.